src/Sulu/Bundle/SecurityBundle/Resources/js/views/Form/tests/toolbarActions/EnableUserToolbarAction.test.js

```diff
@@ -130,7 +127,10 @@ test('Call ResourceRequester with correct parameters when button is clicked', ()
     // $FlowFixMe
     toolbarAction.resourceFormStore.resourceStore.locale = 'de';
 
-    toolbarAction.getToolbarItemConfig().onClick();
+    const toolbarItemConfig = toolbarAction.getToolbarItemConfig();
+    expect(toolbarItemConfig).not.toBeFalsy();
+    // $FlowFixMe
```

Comment:
What I usually prefer instead the expect and the `$FlowFixMe` is something like this:

```javascript
if (!toolbarItemConfig) {
    throw new Error('The ToolbarItemConfig should not undefined or null');
}
```

Still shows a nice error message when running jest and flow does not complain at all :slightly_smiling_face: